.github/PRIVACY.md

```diff
@@ -0,0 +1,51 @@
+# Kindly Security and GDPR Policy
+
+At Kindly, the security of our codebases and the privacy of our community are of paramount importance. This document outlines our policies on security and adherence to the General Data Protection Regulation (GDPR) principles, details how to report security vulnerabilities, and explains our approach to ensuring data privacy.
```

Comment:
`At Kindly ...`

Shouldn't speak as 'Kindly' since that is only a tool - developed under the management of enBloc for Trafalgar Girls.

I would go straight to "The security of our codebase and the privacy of our community..."